bundle/artifacts/upload.go

```diff
@@ -57,12 +58,18 @@ func (m *cleanUp) Apply(ctx context.Context, b *bundle.Bundle) diag.Diagnostics
 		return diag.FromErr(err)
 	}
 
-	b.WorkspaceClient().Workspace.Delete(ctx, workspace.Delete{
-		Path:      uploadPath,
-		Recursive: true,
-	})
+	client, err := getFilerForArtifacts(b.WorkspaceClient(), uploadPath)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	// We intentionally ignore the error because it is not critical to the deployment
+	err = client.Delete(ctx, ".", filer.DeleteRecursively)
```

Comment:
I know this is no different from what we used to do. But still, it makes me uneasy (not to mention on volumes it will take forever if the underlying directory happens to be large). Let's figure out a way to remove this call.